ecology.yaml

```diff
@@ -471,6 +479,31 @@ tools:
     owner: devteam
     tool_panel_section_label: 'FASTA/FASTQ'
 
+
+  - name: xarray_metadata_info
+    owner: ecology
+    tool_panel_section_label: 'GIS Data Handling'
+
+  - name: xarray_select
+    owner: ecology
+    tool_panel_section_label: 'GIS Data Handling'
+
+  - name: xarray_coords_info
+    owner: ecology
+    tool_panel_section_label: 'GIS Data Handling'
+
+  - name: xarray_netcdf2netcdf
+    owner: ecology
+    tool_panel_section_label: 'GIS Data Handling'
+
+  - name: xarray_mapplot
+    owner: ecology
+    tool_panel_section_label: 'GIS Data Handling'
+
+  - name: bwa
```

Comment:
we can not have the same tool in multiple categories :(

Can we remove this? You would need to include the mapping section in your subdomain.

Reply:
yes, for sure! an error on my side I guess